common/src/main/resources/theme/globals/site.variables

```diff
@@ -0,0 +1,3 @@
+@floatingShadow:
```

Comment:
Is this overriding the theme right? I don't see it being used in the css

Reply:
This seems to be an oversight in the "slate" theme. This style is applied when you have a `Segment` that is `raised`. In the default theme, which uses light backgrounds, that's a dark shade around the segment. The "slate" theme didn't change this, and since its background is dark, it was not visible. I tried to make it look similar to the `Card` `raised` style.

However, this will stop working if we switch to a theme with a light background. Should we move this to the slate theme? Is that copy of the theme definitive or is there a chance that we may copy a newer version of the theme over?

Reply:
I doubt we'll update themes

Reply:
Ok, I'll move this to the slate theme then.